src/main/java/net/sf/jabref/model/entry/Author.java

```diff
@@ -74,7 +74,7 @@ public static String addDotIfAbbreviation(String name) {
 
             if(currentChar == '.') {
                 // A.A. -> A. A.
-                if(i + 1 < name.length() && Character.isUpperCase(name.charAt(i + 1))) {
+                if(((i + 1) < name.length()) && Character.isUpperCase(name.charAt(i + 1))) {
```

Comment:
That is an automatic Eclipse addition... All brackets or outer brackets? Outer brackets I find quite useful, but I agree that the ones surrounding i + 1 could be skipped. There are even some more annoying cases of these brackets, e.g. (or a quite similar case)

```
j = k - l + m;
```
becomes
```
j = (k - l) + m;
```